prow/config/config.go

```diff
@@ -2430,6 +2442,10 @@ func parseProwConfig(c *Config) error {
 		c.Gerrit.RateLimit = 5
 	}
 
+	if err := setAllowedPresubmitTriggerRegex(&c.Gerrit); err != nil {
```

Comment:
If you want to start using a function to organize things, please include the other `Gerrit` struct finalization in the function as well rather than just setting the regex. A member function/method of the `Gerrit` struct would also be more appropriate than passing in the struct.